event-app-api/server.js

```diff
@@ -2,10 +2,38 @@ import express from "express";
 import cors from "cors";
 import { sequelize } from "./database.js";
 import { User, Event } from "./models/index.js";
+import bcrypt from "bcrypt";
+
+import bodyParser from "body-parser";
+import cookieParser from "cookie-parser";
+import session from "express-session";
+
+const saltRounds = 10;
 
 const app = express();
 app.use(express.json());
-app.use(cors());
+app.use(
+  cors({
+    origin: "http://localhost:5173",
+    credentials: true,
+    optionSuccessStatus: 200,
+  })
+);
+
+app.use(cookieParser());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use(
+  session({
+    key: "userId",
+    secret: "subset",
+    resave: false,
+    saveUninitialized: false,
+    cookie: {
+      expires: 60 * 60 * 24,
+    },
+  })
+);
 
 app.get("/", (req, res) => {
   res.send("Hello, World!");
```

Comment:
Just curious is this endpoint getting used anywhere? If not you can just remove it